src/compile/selection/transforms/inputs.ts

```diff
@@ -0,0 +1,57 @@
+import {TransformCompiler} from './';
+import {stringValue} from '../../../util';
+import {warn} from '../../../log';
+
+const inputBindings:TransformCompiler = {
```

Comment:
@kanitw Do we want classes instead of const objects? I think it should be possible to still put the compilers in the `compilers` dict.